libraries/opentsdb_instance.rb

```diff
@@ -246,7 +247,7 @@ def action_enable
               group new_resource.group
               cookbook new_resource.cookbook
               notifies :restart, new_resource, :delayed
-              mode 0644
```

Comment:
Rubocop didn't like the leading 0 and wanted `0o644`. I wasn't sure if that was supported in chef, but I know that strings are.